src/graph/validator/FetchEdgesValidator.cpp

```diff
@@ -152,8 +152,7 @@ Status FetchEdgesValidator::validateYield(const YieldClause *yield) {
   auto pool = qctx_->objPool();
   auto *newCols = pool->add(new YieldColumns());
   for (auto col : yield->columns()) {
-    if (ExpressionUtils::hasAny(col->expr(),
```

Comment:
Now support path build expression?

Reply:
yes, pathbuildExpression Is used internally, and the user cannot construct it